packages/core/src/connectors/aliyun-dm/index.ts

```diff
@@ -16,3 +23,43 @@ export const metadata: ConnectorMetadata = {
       '邮件推送（DirectMail）是款简单高效的电子邮件群发服务，构建在阿里云基础之上，帮您快速、精准地实现事务邮件、通知邮件和批量邮件的发送。',
   },
 };
+
+interface Template {
+  type: keyof EmailMessageTypes;
+  subject: string;
+  content: string; // With variable {{code}}, support HTML
+}
+
+export interface AliyunDmConfig {
+  accessKeyId: string;
+  accessKeySecret: string;
+  accountName: string;
+  fromAlias?: string;
+  templates: Template[];
+}
+
+export const sendMessage: EmailSendMessageFunction = async (address, type, data) => {
+  const config: AliyunDmConfig = await getConnectorConfig<AliyunDmConfig>(
+    metadata.id,
+    metadata.type
```

Comment:
since now we only use `id` for identifying, should the function reflects this change?

Reply:
I fixed this in my PR, maybe ignore this.

Reply:
Thanks.